### ISSUE TITLE
HOOBS completely unresponsive. 

### ISSUE THREAD

**Issue by moamick** [hoobs-org/HOOBS#488]
**Description**
Had many issues prior to 3.0. Was unable to update to 3.0 and now HOOBS seems bricked? Now, accessing HOOBS.local returns a “server stopped responding error”. Thoughts on how to repair? 

The unit is plugged in, both to the router and the wall. The lights come on. Nothing has changed with our internet settings. Is there a way to plug the HOOBS into a computer and start over from square one?

I don’t have access to any of the info below because I can’t access the device. 

**Version**
Please include the version of HOOBS you are using.

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [ ] No

**Pervious version**
If you upgraded, please let us know your previous version.

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ ] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [ ] Custom Install

**If this is a custom install, what's your operating system?**
Please let us know what operating system and version you are using. Ex. Fedora 30 or macOS Catalina.

**List your plugins**
Please include a list of the plugins you are using. We need to be able to find it on NPM. If the plugin is no published, please include a link to the repository.

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
POST CONFIG HERE
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
POST LOG HERE
```

**Additional context**

**Comment by mkellsy** [comment]
You might not be getting to the device. if http://hoobs.local doesn't work, try http://hoobs.lan or http://hoobs. If none of that works try using the IP address http://192.168.x.x

**Comment by moamick** [comment]
Just tried this. Nothing works. IP address and hoobs.local just perpetually load, never times out or ever loads anything.

**Comment by mkellsy** [comment]
Can you ssh into the device.
`ssh hoobs@[IP address]` Replace [IP ADDRESS] with the ip address of the device.

The password is hoobsadmin.
Once you are in there run this command
```
sudo hoobs service log
```
Post the results.

**Comment by BobbySlope** [comment]
@moamick please provide the information requested by @mkellsy 
otherwise we can't solve the issue
closing for the moment, reopen it, as soon as we get the logs

**Comment by moamick** [comment]
I reached out to HOOBS support. I had to take the device apart and flash the micro SD. 

It’s been resolved.

**Comment by Fr-nkS** [comment]
I seem to have a similar issue. Bought hoobs on micro SD card, set up account for an hour, now devices connected yet, learned that the Mac desktop app only works with hoobs version 4, updated to version 4 --> cannot connect to the Raspi through http://[IP address] anymore.

Tried ssh as advised above. was about to share the logs but got stuck (see screenshot)
![screenshot_2021_07_18-16_07_49](https://user-images.githubusercontent.com/87608836/126070373-30eaa3fb-a7ea-4798-a394-fc42967ab75f.jpg)

How do I "enable the Hoobs service"?

I was hoping to spare me all the fiddling just purchasing an out-of-the-box system. In case I gonna end up flashing the SD card, I would go full circle backwards... Anything obvious I did wrong on my end?
